docs/RegistrationChecklist.md

```diff
@@ -0,0 +1,58 @@
+# Minimum Ontology Registration Request Checklist
+
+The _Minimum Ontology Registration Request Checklist_ (MORRC) is intended to provide a first pass to fairly evaluate ontology registration requests to the OBO foundry. 
+The goal is to formalize what is currently an ad-hoc process, to filter out ontologies that are clearly out of scope for OBO. It is intentionally weaker than full
+conformance, because (a) we want to allow registration of imperfect ontologies (b) partial conformance is easier to quickly and objectively check.
+
+Note this does not change the 2 week rule. Anyone from the OBO community is free to object to the ontology on the basis of any of these checks. 
+However, if no objections are raised in the 2 week period, then the ontology becomes registered. Once registered an ontology is never unregistered.
```

Comment:
In the Operations call, we discussed that two week the rule is too tight. I suggest changing to "two weeks from next operations call" or "four weeks rule".